packages/docs/docs/.vuepress/nav/en.js

```diff
@@ -4,53 +4,31 @@ module.exports = [
     link: "/guide/"
   },
   {
-    text: "Config Reference",
-    link: "/config/"
-  },
-  {
-    text: "Plugin",
-    link: "/plugin/"
-  },
-  {
-    text: "Theme",
-    link: "/theme/"
+    text: "Plugins / Themes",
+    link: "/market/"
   },
   {
     text: "Learn More",
     items: [
       {
-        text: "API",
-        items: [
-          {
-            text: "CLI",
-            link: "/api/cli.html"
-          },
-          {
-            text: "Node",
-            link: "/api/node.html"
-          }
-        ]
+        text: "Config reference",
+        link: "/config/index.html"
+      },
+      {
+        text: "Api",
```

Comment:
Should we be specific here that it is the CLI API?

Reply:
not really actually i didnt do a lot of stuff i wanted first to present the idea behind